src/azul/azulclient.py

```diff
@@ -106,10 +107,13 @@ def test_notifications(self, test_name: str, test_uuid: str) -> Tuple[List[dict]
         effective_bundle_fqids = set()
         for bundle_fqid in real_bundle_fqids:
             new_bundle_uuid = str(uuid.uuid4())
-            _, _, version = bundle_fqid.partition('.')
-            effective_bundle_fqids.add(new_bundle_uuid + '.' + version)
+            # Using a new version helps ensure that any aggregation will choose this entity.
+            # Failing to do this before caused https://github.com/DataBiosphere/azul/issues/1174
+            new_bundle_version = datetime.utcnow().isoformat().replace(':', '') + 'Z'
```

Comment:
Can you consolidate with this:

https://github.com/DataBiosphere/azul/blob/develop/scripts/copy_bundles.py#L227

by moving that code into the `azul.dss` module and reusing it here. You may have to add that module to the suite in `test_doctest.py`.

Just double check that the code I link to actually generates a version representing the current time in UTC.

Reply:
I missed
 > by moving that code into the azul.dss module and reusing it here. You may have to add that module to the suite in test_doctest.py.

On my first pass. It was simpler and seemed appropriate to put it in a separate PR which I did: #1363.